src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -502,7 +502,7 @@ public int countApplicationUpdatedMessages() {
      */
     private File getMessagesLogFile(ServerTask serverTask) {
         File logFile;
-        if (serverTask != null) {
+        if (!container && serverTask != null) {
```

Comment:
So I think if container is true, we always want to not use the serverTask. And if container is not true, we always want to use serverTask. Right? I would rewrite as follows:
```
if (container) {
   logFile = new File(serverDirectory, "logs/messages.log");
} else {
   logFile = serverTask.getLogFile();
}
```

Reply:
Fixed